releases: proceed to final tag as fast as possible (#1814)
sourcegraph/about: handbook/engineering/releases/release_issue_template.md

  yarn run release changelog:cut $MAJOR.$MINOR.0
  ```
- [ ] Create the `$MAJOR.$MINOR` branch off the CHANGELOG commit in the previous step: `git branch $MAJOR.$MINOR && git push origin $MAJOR.$MINOR`.

> ## Release candidates

Reviewer: ```suggestion
```

As it stands this makes me think I might do this on the 20th instead of immediately after branch cut, so I suggest removing this header.

> Upon branch cut, cut and test the first release candidate:

Reviewer: ```suggestion
Upon branch cut, create and test the first release candidate:
```

Author: Added this suggestion, but we seem to use a mix of "cut" and "create" for creating releases - probably not very important to unify the terminology though I suppose

  - [ ] New Sourcegraph Kubernetes cluster:
    - Run the initializer on a new Sourcegraph Kubernetes cluster.
    - Run the regression test suite.

> - [ ] File any regressions as `release-blocker` issues.

Reviewer: ```suggestion
- [ ] File any regressions as `release-blocker` issues and assign the appropriate teams.
```

> Revert or disable features that may cause delays. As necessary, `git cherry-pick` bugfix (not feature!) commits from `main` into the release branch. Continue to cut new release candidates daily as necessary, until no more `release-blocker` issues remain:

Reviewer: ```suggestion
Revert or disable features that may cause delays. As necessary, `git cherry-pick` bugfix (not feature!) commits from `main` into the release branch. Continue to create new release candidates daily or as necessary, until no more `release-blocker` issues remain:
```

> Aggressively revert or disable features that may cause delays. As necessary, `git cherry-pick` bugfix (not feature!) commits from `main` into the release branch. Continue to cut new release candidates daily as necessary, until no more `release-blocker` issues remain:

Reviewer 2: ```suggestion
Revert or disable features that may cause delays. As necessary, `git cherry-pick` bugfix (not feature!) commits from `main` into the release branch. Continue to cut new release candidates daily as necessary, until no more `release-blocker` issues remain:
```

- [ ] Cut the Nth release candidate:
  ```
  N=<release-candidate-number> yarn run release release:create-candidate $MAJOR.$MINOR.0-rc.$N
  ```
- [ ] Re-run the automated test suite against the new release candidate, file any regressions as
  `release-blocker` issues.
  - [ ] If necessary, manually test features or workflows affected by the cherry-pick.

> - [ ] Post a release status update to Slack - review all release-blocking issues, and ensure someone is resolving each. If there are no more release-blocking issues, proceed to the next section.

Reviewer: ```suggestion
- [ ] Post a release status update by running the command below. Ensure someone is resolving each release-blocking issue. If there are no more release-blocking issues, proceed to tag the final release in the next section.
```
